Material.Styles/MaterialToolKit.xaml

```diff
@@ -20,11 +20,6 @@
     <StyleInclude Source="avares://Material.Styles/SnackbarHost.xaml"/>
     <StyleInclude Source="avares://Material.Styles/MaterialInternalIcon.axaml"/>
     
```

Comment:
leave a comment on this line about dialog host supports is recommended I think

Reply:
Hmm, good idea

Reply:
like `<!-- dialog host style have been removed from index of styling for avoid independend package linking -->` or etc.